app/presenters/summary/components/base_answer.rb

```diff
@@ -16,7 +16,7 @@ def initialize(question, value, *args)
       end
 
       def show?
-        show.presence || value?
```

Comment:
`presence` doesn't work if `show` is [false](url), as `false.present? `resolves to `false`, so `show?` didn't get set as `false`, but as `value?`